server/src/main/java/org/jboss/as/server/parsing/StandaloneXml.java

```diff
@@ -528,16 +527,16 @@ private void parseHttpManagementInterface1_0(final XMLExtendedStreamReader reade
                 final Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
                 switch (attribute) {
                     case INTERFACE: {
-                        HttpManagementResourceDefinition.INTERFACE.parseAndSetParameter(value, mgmtSocket, reader);
+                        //ignore
                         hasInterfaceName = true;
```

Comment:
Shouldn't this fail?

AIUI, parse will pass, since hasInterfaceName=true, but presumably it will fail later due to no socket-binding setting.

The upshot of all this is a 1.0 config is likely useless, since there's no socket-binding alternative available. But we should fail with a descriptive failure message.